tasks.py

```diff
@@ -1,76 +1,77 @@
 from invoke import task
 from invoke.exceptions import Failure
+from invoke.tasks import call
+
+
+def run_or_fail(c, cmd, fail_message="Error", fail_callback=None):
+    try:
+        cmd(c)
+    except Failure as ex:
+        print(fail_message)
+        if fail_callback:
+            fail_callback(c)
+        exit()
```

Comment:
This approach is similar to the one you suggested to me, but I preferred to pass a function rather than a command to have the possibility to pass parameters to the function without having to add other input parameters
I have also foreseen the possibility of passing a callback in case of exception